01/app.js

```diff
@@ -11,7 +11,7 @@ function addUser(e) {
     if (firstName.length === 0) {
         alert('First Name is required!');
     }
-    if (lastName.length === 0) {
+    else if (lastName.length === 0) {
```

Comment:
Teraz jest ok, ale lepiej jest wykonywać tego typu warunki w oparciu o `const errors = []`.
Jeśli imie i nazwisko będzie błędne to najpierw użytkownik dostanie info o błędnym imieniu, a już już ono będzie ok to wtedy dopiero będzie informacja o błędnym nazwisku. Najlepiej informować o wszystkich błędnych od razu :)

Reply:
OK.